webhook/resourcesemantics/validation/reconcile_config.go

```diff
@@ -100,23 +100,51 @@ func (ac *reconciler) reconcileValidatingWebhook(ctx context.Context, caCert []b
 	logger := logging.FromContext(ctx)
 
 	rules := make([]admissionregistrationv1.RuleWithOperations, 0, len(ac.handlers))
-	for gvk := range ac.handlers {
+	for gvk, config := range ac.handlers {
 		plural := strings.ToLower(flect.Pluralize(gvk.Kind))
 
+		// If SupportedVerbs has not been given, provide the legacy defaults
+		// of Create, Update, and Delete
+		supportedVerbs := []admissionregistrationv1.OperationType{admissionregistrationv1.Create,
```

Comment:
nit: Can you line-break this?  I think it could be easy to misread this as just Update/Delete with current wrapping.

Reply:
yep, done!